wmn-data.json

```diff
@@ -2393,6 +2393,16 @@
         "known" : ["carlos01", "paul"],
         "cat" : "hobby"
        },
+       {
+        "name" : "Holopin",
+        "uri_check" : "https://holopin.io/@{account}#",
+        "e_code" : 200,
+        "e_string" : "Holobytes",
+        "m_code" : 200,
+        "m_string" : "Not found",

```

Comment:
Let's make this `m_string` more robust by making it `Not found</div>`

Reply:
Done.